executor/chroot/chroot_executor_test.go

```diff
@@ -101,12 +100,9 @@ func Test(t *testing.T) {
 						Entrypoint: []string{"ls", "/data/test"},
 					}
 
-					job, _ := executor.Run(formula)
+					job := e.Start(formula)
 					So(job, ShouldNotBeNil)
-					So(job.ExitCode(), ShouldEqual, 0)
-					msg, err := ioutil.ReadAll(job.OutputReader())
-					So(err, ShouldBeNil)
-					So(string(msg), ShouldEqual, "1\n2\n3\n")
```

Comment:
We need these back.  This doesn't actually test anything (except directory existence, which is different than an input having *worked*) without this check.